tls/provider.go

```diff
@@ -19,9 +19,12 @@ import (
 
 // CertificateProvider describes an interface by which TLS certificates may be managed.
 type CertificateProvider interface {
-	// GetCA returns the active root CA.
+	// GetCA returns the active root CA certificate.
 	GetCA() ([]byte, error)
 
+	// GetCACertPool returns the active root CA certificate a certificate pool.
```

Comment:
```suggestion
	// GetCACertPool returns the active root CA certificate of a certificate pool.
```

unless that's what was expected

Reply:
should be `as`, but thanks